src/scene/fizzler.c

```diff
@@ -17,9 +19,16 @@
 #define GFX_PER_PARTICLE(particleCount) ((particleCount) + (((particleCount) + 7) >> 3) + 1)
 
 void fizzlerTrigger(void* data, struct CollisionObject* objectEnteringTrigger) {
+	struct Fizzler* fizzler = (struct Fizzler*)data;
+	
     if (objectEnteringTrigger->body) {
         objectEnteringTrigger->body->flags |= RigidBodyFizzled;
     }
+	
+	int decorType = decorIdForObjectDefinition((struct DecorObjectDefinition*)objectEnteringTrigger->collider);
```

Comment:
This is a hack that only works since the collider is the first member in the `DecorObjectDefinition` struct.

It's how triggers do this check too (which is where you probably got it from), and not your hack to fix, but something to be aware of.